pkg/operator/ceph/csi/spec.go

```diff
@@ -428,3 +433,51 @@ func createCSIDriverInfo(clientset kubernetes.Interface, name string) error {
 
 	return err
 }
+
+// ValidateCSIVersion checks if the configured ceph-csi image is supported
+func ValidateCSIVersion(clientset kubernetes.Interface, namespace, csiPluginImage, rookImage, serviceAccountName string) error {
```

Comment:
@travisn do we also need to sent back the supported version in error message? as this is required to recorrect the value in the yaml file?

Reply:
If it was possible to parse the version of the configured CSI image it will get logged already here [`ValidateCSIVersion`](https://github.com/shaas/rook/blob/master/pkg/operator/ceph/csi/spec.go#L480)

Reply:
the link you posted is printing the detected version, not the supported version which we are adding at https://github.com/shaas/rook/blob/master/pkg/operator/ceph/csi/version.go#L36 right? in case if it's not supported we log the supported version along with unsupported version it helps users to correct the deployment.

Reply:
Ah Correct and good point. I will adopt the message accordingly.